templates/flavors/custom-builds/patches/custom-builds.yaml

```diff
@@ -0,0 +1,142 @@
+apiVersion: controlplane.cluster.x-k8s.io/v1alpha4
+kind: KubeadmControlPlane
+metadata:
+  name: "${CLUSTER_NAME}-control-plane"
+  annotations:
+    controlplane.cluster.x-k8s.io/skip-kube-proxy: "true"
+spec:
+  kubeadmConfigSpec:
+    useExperimentalRetryJoin: true
+    clusterConfiguration:
+      kubernetesVersion: "${KUBERNETES_VERSION}"
+    postKubeadmCommands:
+    - bash -c /tmp/kubeadm-bootstrap.sh
```

Comment:
this filename is misleading, we're not actually bootstrapping kubeadm since this is running after kubeadm init/join runs

Reply:
thx, changed